src/libs/getPlaidLinkTokenParameters/types.ts

```diff
@@ -0,0 +1,14 @@
+type PlaidLinkTokenParameters = {
+    // eslint-disable-next-line @typescript-eslint/naming-convention
```

Comment:
NAB but why are we suppressing this several times in this PR? Would it make sense to abide by this convention as part of this PR or would that require a lot of updated that are out of scope for this PR?

Reply:
@cead22 This convention is more of an exception for this specific case, so I think we shouldn't apply it everywhere.

Reply:
Can you help me understand this a little better? I don't think it's truly an exception since there's 5 occurrences in this PR, which is very small.

What are the reasons for not updating the variable names to meet the naming convention?

Reply:
@cead22 These variables are parameters that are later sent to the API
[API.read('OpenPlaidBankLogin', params, {optimisticData});](https://github.com/Expensify/App/blob/main/src/libs/actions/Plaid.js#L36C5-L36C5)

So even if we rename it here, then before sending it to the API, we will need to do something like `{redirect_uri: redirectUri}`, and it also can cause a convention warning

Reply:
That makes sense, so I think the correct way to complete this issue is
- Update the backend to accept redirectURI and redirect_uri (cc @nkuoch)
- Update JS to use the right naming convention

If we don't do this, then we'll continue breaking our conventions forever, and continue adding exceptions, which defeats the purpose of having them.

We need to think about our code holistically and continuously update the code surrounding the code we edit

Reply:
I agree with @cead22, however I don't think it's in the scope for this PR. 

Once majority of files are migrated to typescript we can search for these occurrences and fix these names more easily - TS will warn us if names are migrated and used correctly. What do you think? @cead22 @VickyStash

Reply:
I agree with @cead22 and I would avoid pushing this off into another PR. I did something similar with `filename` and `receipeFilname` recently. 

I understand it's not strictly in the scope of this PR, but If we wrapped every PR with a second PR that was basically "clean up all the code around the previous PR X" I think that would get out of hand.

In the interest of always leaving code in a better state than it was found, I think it should be cleaned up here.

If it's a matter that ends up significantly increase your workload, we are pretty fair and can increase compensation using our discretion, so I wouldn't worry too much about that (if it was at all part of your thinking).

Reply:
@tgolen @cead22 So, let's hold this PR and create another GH issue to change those parameters in the backend. Once that issue is done we can go back to this one and finish it. Sounds good? Do you want me to create the GH issue for you?

Reply:
I made the [GH issue](https://github.com/Expensify/Expensify/issues/318749) (in E/Expensify). I can post here when the change to accept both redirect_uri and redirectURI is live

Reply:
Thanks @cead22 🙌

Reply:
PR is merged. It should be live on Tuesday

Reply:
@cead22 Got it, thank you!

Reply:
@cead22 Just want to confirm, have you also updated `android_package` field to be accepted as both `android_package` and `androidPackage`?

Reply:
🤦 I did not, I'll do that now

Reply:
Yes, they were deployed to production yesterday!

Reply:
@cead22 Thank you! I've updated the PR